src/absil/ilreflect.fs

```diff
@@ -1500,20 +1500,46 @@ let emitParameter cenv emEnv (defineParameter: int * ParameterAttributes * strin
 //----------------------------------------------------------------------------
 // buildMethodPass2
 //----------------------------------------------------------------------------
-  
+
+#if !FX_RESHAPED_REFEMIT || NETCOREAPP3_0
+
+let enablePInvoke = true
+
+#else
+
+// We currently build targeting netcoreapp2_1, and will continue to do so through this VS cycle
+// but we can run on Netcoreapp3.0 so ... use reflection to invoke the api, when we are executing on netcoreapp3.0
+let definePInvokeMethod =
+    typeof<TypeBuilder>.GetMethod("DefinePInvokeMethod", [|
+        typeof<string>;
```

Comment:
These semicolons aren't necessary

Reply:
@cartermp you are quite correct.  Initially it was a 500 character single line, when I tidied that up, I didn't complete the job.